frontend-shared/src/test/svg-icon-test.js

```diff
@@ -0,0 +1,93 @@
+import { createElement, render } from 'preact';
+
+import { SvgIcon, availableIcons, registerIcons } from '../svg-icon';
+
+describe('SvgIcon', () => {
+  // Tests here use DOM APIs rather than Enzyme because SvgIcon uses
+  // `dangerouslySetInnerHTML` for its content, and that is not visible in the
+  // Enzyme tree.
+
+  // Global icon set that is registered with `SvgIcon` outside of these tests.
+  let savedIconSet;
+
+  beforeEach(() => {
+    savedIconSet = availableIcons();
+
+    registerIcons(
+      {
+        'arrow-left': require('../../images/icons/arrow-left.svg'),
```

Comment:
These tester svgs are different. I picked icons that I thought would be more appropriate for a shared lib.